### DISCUSSION TITLE
Subscription for API account

### DISCUSSION THREAD

**Discussion by jaydenlzx** [iTwin/itwinjs-core#5035] [General]
Hello I would like to check if Bentley has introduced a subscription model for API account. I suddenly couldnt run any of my script using API account.

**Comment by calebmshafer** [comment]
@jaydenlzx are you receiving an error when contacting the iTwin platform services? If so, would you please share the error you're hitting.

**Reply by jaydenlzx** [reply]
I have been getting 401 error.

**Reply by arnab-git-hub** [reply]
Can you please provide the following to better understand the issue:
1. The whole signature of the API and the API URL link which you are trying to access.
2.  The http responses
3. Client ID

**Reply by jaydenlzx** [reply]
![image](https://user-images.githubusercontent.com/80751784/217751297-feaec853-5fba-4683-a2cb-b67e6cc73b9f.png)

client_id = "service-ic9GskfCxPWYu4NeVGxAVGZ75"

Is there a subscription required? 

I am trying to GET Project Form Data

**Reply by arnab-git-hub** [reply]
Looks like your trial has expired. Can you please provide your email id to confirm the same?

**Reply by jaydenlzx** [reply]
my email ID is Jayden_lim@optimus-pw.com.

**Reply by jaydenlzx** [reply]
![image](https://user-images.githubusercontent.com/80751784/217794376-e9b4e1e5-56e4-42c9-aa99-9392b3915d14.png)
I went to check my subscription for this client ID. It shows no subscription.

**Reply by arnab-git-hub** [reply]
Please visit https://developer.bentley.com/ and click on the "Free Trial" button on the top right corner and sign up. It will start your 90 day trial.

![image](https://user-images.githubusercontent.com/78776295/217797877-7077ece6-1634-41fd-91e4-5b0664df4c85.png)

**Reply by jaydenlzx** [reply]
I have already requested for free trial. So am i right to say that after 90 days of free trial, i will not be able to use this api unless i have some form of subscription model.

**Reply by arnab-git-hub** [reply]
We noticed a workflow when individual user registers with his email, starts a trial, but later gets pulled into an organization. From the screenshot we can see that you are part of an organization, in this case JTC Corporation. 

Once the user is part of organization, his personal subscription is not longer applicable and subscription information is being pulled from organization level.

We have seen that for JTC Corporation has no active subscription and hence they are eligibale for organizational free trail. They can start this organization level trial with a free trial button.

If you find issues, please provide step by step screenshot.

**Reply by arnab-git-hub** [reply]
@jaydenlzx Can you please provide an update if the issue has been resolved? Thanks
